seed/data_importer/tasks.py

```diff
@@ -1620,12 +1621,15 @@ def pair_new_states(merged_property_views, merged_taxlot_views):
 def _validate_use_cases(file_pk, progress_key):
     import_file = ImportFile.objects.get(pk=file_pk)
     progress_data = ProgressData.from_key(progress_key)
+    bs = BuildingSync()
+    bs.import_file(import_file.file)
```

Comment:
Yet another gotcha, we allow users to upload zip files of BuildingSync files, so this causes the BuildingSync class to try and parse a zip. We might have to open the zip, get the version of the first file and validate them all as that version... What do you think about that? Alternatively we could iterate through all the files and validate them individually

Reply:
I honestly feel we should validate all the files if a ZIP is presented.  I'll refactor to accomodate!